elliottlib/util.py

```diff
@@ -258,3 +258,15 @@ def convert_remote_git_to_https(source):
         string=source.strip(),
     )
     return re.sub(string=url, pattern=r'\.git$', repl='').rstrip('/')
+
+
+def minor_version_tuple(bz_target):
+    """
+    Turns '4.5' or '4.5.z' into numeric (4, 5)
+    Assume the target version begins with numbers 'x.y' - explode otherwise
+
+    :param bz_target: A string like "4.5.0"
+    :return: A tuple like (4, 5)
+    """
+    major, minor, _ = f"{bz_target}.z".split('.', 2)
```

Comment:
Sometimes `bz_target` value might come as `---`, which can cause the following exception:
```
    major, minor, _ = f"{bz_target}.z".split('.', 2)
ValueError: not enough values to unpack (expected 3, got 2)
```
```suggestion
    if bz_target == '---':
        return (0, 0)
    major, minor, _ = f"{bz_target}.z".split('.', 2)
```

Reply:
after applying the suggestion above, I managed to run the new command:
```
$ elliott -g openshift-4.6 verify-attached-bugs
2020-11-11 13:37:43,670 INFO Cloning config data from https://github.com/openshift/ocp-build-data.git
2020-11-11 13:37:45,388 INFO Using branch from group.yml: rhaos-4.6-rhel-8
Retrieving bugs for advisory 65846
Retrieving bugs for advisory 65844
Retrieving bugs for advisory 65845
Found 40 bugs
Regression possible: bug 1885667 is a backport of bug 1883444 which has status ON_QA
Some bug problems were listed above. Please investigate.
```

Reply:
nice catch, thanks!